build.gradle

```diff
@@ -6,9 +6,9 @@ plugins {
     id 'maven-publish'
     id "io.github.gradle-nexus.publish-plugin" version "1.3.0"
     id 'signing'
-    id 'org.jetbrains.kotlin.jvm' version "1.6.21"
-    id 'com.github.johnrengelman.shadow' version '4.0.2'
-    id "org.owasp.dependencycheck" version "8.2.1"
+    id 'org.jetbrains.kotlin.jvm' version "1.8.22"
```

Comment:
Maybe complete th2-gradle-plugin and spread it instead of updating the same things in each repo

Reply:
It would be good of course, but it is not yet completed) I will try to find time to look at the plugin